tools/test.py

```diff
@@ -128,7 +132,8 @@ def main():
     # build the dataloader
     dataset = build_dataset(cfg.data.test, dict(test_mode=True))
     dataloader_setting = dict(
-        videos_per_gpu=cfg.data.get('videos_per_gpu', 2),
+        videos_per_gpu=cfg.data.get('videos_per_gpu', 1)
+        if args.multi_batches else 1,
         workers_per_gpu=cfg.data.get('workers_per_gpu', 0),
```

Comment:
Better to set the default value of 'workers_per_gpu' as 1 (for better testing speed).

Reply:
Would higher value be better?